docs/data/data-grid/prompt/PromptWithDataSampling.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  GridToolbarPromptControl,
+  GridToolbar,
+  GridToolbarContainer,
+} from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import Stack from '@mui/material/Stack';
+
+const REMOTE_ENDPOINT =
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost'
+    : 'https://mui.com/x/api/data-grid';
```

Comment:
This is now completely removed and only shown in the docs as a suggestion, since I have updated API of the component to be more aligned with the data source for the grid (having a callback, rather than just passing an url)